docs/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
+- Command for manual syncing Jira metadata (OSIDB-3219)
```

Comment:
IMHO this one is technical

Reply:
I skimmed through and we have these commands in the changelog from the past, so I just followed status quo.

Reply:
Ok, then up to you.